source/mongodb/data-access.txt

```diff
@@ -0,0 +1,113 @@
+.. _client-data-access:
+
+============================
+Query Atlas from Client Apps
+============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can query data in linked MongoDB Atlas clusters from client applications
+using standard MongoDB query language (MQL) syntax with **MongoDB Data Access**
+in the Realm SDKs.
+
+The MongoDB Data Access secures data with role-based App Services rules, schemas,
+and filters. These features let construct complex client-side queries with MQL
+while your data remains secure on the server.
+
+When to Use MongoDB Data Access
+-------------------------------
+
+Some general scenarios when you might want to use MongoDB Data Access in an application:
+
+- You want to use an API like the MongoDB drivers
+- You want to perform aggregations
+- You want to dynamically construct queries from the client device.
+
+You might want to use MongoDB Data Access **instead of** :ref:`Atlas Device Sync <sync>`
+in an application in these scenarios:
+
+- You are querying documents that are not or cannot be modeled in Realm Database.
+- A non-Realm service generates collections that you want to access.
+- You do not want to persist data locally.
+- You are using the Realm Web SDK, which does not support Realm Database or Device Sync.
+
+As alternative to MongoDB Data Access in **client-side** scenarios,
+you can use the :ref:`Atlas Data API <data-api>` or the :ref:`Atlas GraphQL API <graphql-api>`.
+You can also use these APIs without a Realm SDK for your client app's programming
+language as long as the language has a HTTPS or GraphQL client.
+However, these APIs do not feature the same type-safe MQL syntax
+as the Realm SDKs' MongoDB Data Access.
+
+As alternative to MongoDB Data Access in **server-side** scenarios,
+you can use the Atlas Data API or the Atlas GraphQL API. You can also use
+a standard MongoDB driver. If you would like to apply App Services Rules
```

Comment:
This reads kinda like "On the client, use Data API or GraphQL. On the server, use Data API or GraphQL."

Maybe we could switch up the wording to be a little less repetitive?

Reply:
yea will think bit more about this to make less repetitive.